sys/src/go/internal/ufs/filesystem.go

```diff
@@ -446,15 +440,16 @@ func (e *FileServer) Rwrite(fid protocol.FID, o protocol.Offset, b []byte) (prot
 	return protocol.Count(n), err
 }
 
-func NewUFS(opts ...protocol.ListenerOpt) (*protocol.Listener, error) {
+func NewUFS(root string, debug int, opts ...protocol.ListenerOpt) (*protocol.Listener, error) {
```

Comment:
so ... we can do this later but convention would be to make ALL of these opts ...protocol.ListenerOpt
NewUFS(opts ...protocol.ListenerOpt)
and then we would supply things like
WithRoot(root string)
WithDebug()
and so on.
 Up to you if you want to do that now but it would be nice to get it this way from the start.

OTOH if you really prefer it to look this way, that's fine too, since it kind of matches your command.